resources/web/style.css

```diff
@@ -112,17 +112,16 @@ body {
 }
 
 h1 {
-    font-family: Roboto, Verdana, Arial, sans-serif;
     color: white;
     padding-top: 20px;
     margin-left: 15px;
     font-size: 26px;
-    font-family: arial;
+    font-family: Arial, sans-serif;
     display: inline-block;
 }
```

Comment:
Should it get `font-family: Roboto, Verdana, Arial, sans-serif;`
or only Arial?